demo/api/package.json

```diff
@@ -61,7 +61,7 @@
         "faker": "^5.0.0",
         "file-type": "^16.0.0",
         "graphql": "^15.0.0",
-        "graphql-type-json": "^0.3.0",
+        "graphql-scalars": "^1.23.0",
```

Comment:
Since we're not using it in Demo and it's not a peer dependency, could we remove it?

Reply:
Good point – b69518e1f0e9c186fe6dc98e5bf7a08f17e792a5